src/main/java/ru/sbtqa/tag/pagefactory/support/ScreenShooter.java

```diff
@@ -48,4 +48,26 @@
         }
     }
 
+    /**
+     * Takes screenshot as indicated in application.properties
+     *
+     * @return screenshot in byte array
+     */
+    public static byte[] take() {
+        String screenshotStrategy = Props.get("screenshot.strategy", "raw");
+
+        byte[] screenshot;
+
+        switch (screenshotStrategy) {
+            case "driver":
+                screenshot = ScreenShooter.takeWithDriver();
+                ParamsHelper.addAttachment(screenshot, "Screenshot by Driver", Type.PNG);
```

Comment:
I think, that the main purpose of this class is taking screenshots, thats why we shouldn't do attaching to report here.